src/test/kotlin/no/nav/familie/ef/sak/repository/GrunnlagsdataRepositoryTest.kt

```diff
@@ -47,4 +47,16 @@ internal class GrunnlagsdataRepositoryTest : OppslagSpringRunnerTest() {
         assertThat(behandlinger.find { it.first == behandling.id }!!.second.let { BehandlingStatus.valueOf(it) })
                 .isEqualTo(BehandlingStatus.OPPRETTET)
     }
-}
\ No newline at end of file
+
+    @Test
+    internal fun `bakåtkompatibilitet - tidligereVedtaksPerioder er null då den ikke finnes med i tidligere objekter`() {
+        val fagsak = fagsakRepository.insert(fagsak())
+        val behandling = behandlingRepository.insert(behandling(fagsak))
+
+        val grunnlagsdata = opprettGrunnlagsdata().copy(tidligereVedtaksperioder = null)
```

Comment:
Ble litt usikker - her definerer vi null eksplisitt - jeg var mest spent på der hvor verdien ikke finnes i json-strukturen, men testet dette manuelt og gikk fint :)

Reply:
God poeng, men tror den settes på field-nivå, så den bruker ikke konstruktorn for å sette verdiene i et objekt